libsmtutil/Z3CHCInterface.cpp

```diff
@@ -33,6 +33,13 @@ Z3CHCInterface::Z3CHCInterface(optional<unsigned> _queryTimeout):
 	m_context(m_z3Interface->context()),
 	m_solver(*m_context)
 {
+	Z3_get_version(
```

Comment:
Is this not exposed via the C++ API?

Reply:
Apparently meant to be done like this: https://github.com/Z3Prover/z3/issues/1805